projects/batfish-client/src/org/batfish/client/Client.java

```diff
@@ -380,86 +372,94 @@ static void validateType(JsonNode value, Variable variable)
       switch (expectedType) {
          case BOOLEAN:
             if (!value.isBoolean()) {
-               throw new BatfishException("");
+               throw new BatfishException(
+                     String.format("Value: %s, is not a valid Boolean " +
+                           "value.", value.toString()));
             }
             break;
          case COMPARATOR:
             if (!(COMPARATORS.contains(value.textValue()))) {
-               throw new BatfishException("");
+               throw new BatfishException(
+                     String.format("Value: %s, is not a valid comparator.",
+                           value.toString()));
             }
             break;
          case INTEGER:
             if (!value.isInt()) {
-               throw new BatfishException("");
+               throw new BatfishException(
+                     String.format("Value: %s, is not a valid Integer.",
+                           value.toString()));
```

Comment:
calling `.toString()` on something being passed to `String.format` is not necessary.